### PR TITLE
Feature#35 dashboard

### LINKED ISSUE #35: Create dashboard page
# REQUIREMENTS
We're going to create a dashboard page for admin user

# Acceptance Criteria
- [x] Create dashboard
- [x] Make it functional

### PR DESCRIPTION
**Pull request by xFerchoVJ**
# Description
What did you implemented/Why did you implemented this:
 - I created the dashboard and made it functional

 Example:
  - I created the dashboard and also integrated the routes for management.
  
  ## Screenshots 
image

### CODE REVIEW

**Review by SEVERO96 — APPROVED** [review]

**Review by SEVERO96 — DISMISSED** [review]

**Review by SEVERO96 — DISMISSED** [review]

File: checador/app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
-    
+    <!--<p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>-->
```
Comment (YilKanda) [inline checador/app/views/layouts/application.html.erb]: If you don't occupy it, delete it.

File: checador/app/views/employees/index.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<h2>Manage Employees</h2>
```
Comment (YilKanda) [inline checador/app/views/employees/index.html.erb]: Add a blank line.

File: checador/app/views/employees/_index.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<h1>All employees</h1>
```
Comment (YilKanda) [inline checador/app/views/employees/_index.html.erb]: Add a blank line.

File: checador/app/views/companies/index.html.erb [hunk 4]
```diff
@@ -0,0 +1 @@
+<h1>Company Branches</h1>
```
Comment (YilKanda) [inline checador/app/views/companies/index.html.erb]: Add a blank line.

File: checador/app/views/admin/dashboard.html.erb [hunk 5]
```diff
@@ -1,9 +1,60 @@
...
+              </p>
+        </div>
+
+        <div class="container-fluid mt-5">
+          <%= render 'employees/index'%>
+        </div>
+
+      </div>
+      
+</div>
```
Comment (YilKanda) [inline checador/app/views/admin/dashboard.html.erb]: Add a blank line.

File: checador/app/views/admin/dashboard.html.erb [hunk 6]
```diff
@@ -1,9 +1,60 @@
...
+          </ul>
+        </div>
+      </nav>
+
+      <div class="container">
+        <div class="d-flex justify-content-between align-items-center p-3">
+              <h2 class="margin">Welcome back, Admin!</h2>
+              <p class="m-0 fst-italic text-muted">
+                See the latest information
+              </p>
```
Comment (YilKanda) [inline checador/app/views/admin/dashboard.html.erb]: Check the indentation.

File: checador/app/assets/stylesheets/admin.scss [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  border-bottom: 1px solid #ccc;
+}
+.btn-search {
+  margin-left: 165px;
+}
+
+.margin{
+  margin-top: 15px;
+  margin-bottom: 0px;
+}
```
Comment (YilKanda) [inline checador/app/assets/stylesheets/admin.scss]: Add a blank line.

File: checador/app/assets/stylesheets/admin.scss [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    .logo {
+      padding: 0.875rem 3.25rem;
+    } 
+    .menu {
+        width: 18rem;
+        a{
+          font-size: 17px;
+          text-decoration: none;
+      }
+    }
```
Comment (YilKanda) [inline checador/app/assets/stylesheets/admin.scss]: Check indentation

File: checador/app/assets/stylesheets/admin.scss [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+body{
+    overflow: hidden;
+    font-weight: 300;
+    color: $black;
+}
+
+
+
+.bg.primary { 
+    background-color: $primary-color important;
```
Comment (YilKanda) [inline checador/app/assets/stylesheets/admin.scss]: As mentioned @jaimelr, 

> Avoid the use of !important whenever possible. This makes it hard to reuse styles and can mess up your code later when the codebase grows.

File: checador/app/assets/stylesheets/admin.scss [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+}
+.bg-light {
+  background-color: white important;
```
Comment (YilKanda) [inline checador/app/assets/stylesheets/admin.scss]: As mentioned @jaimelr, 

> Avoid the use of !important whenever possible. This makes it hard to reuse styles and can mess up your code later when the codebase grows.

File: checador/app/views/admin/dashboard.html.erb [hunk 11]
```diff
@@ -1,9 +1,60 @@
...
+            </a>
+            <a href="/employees" class="d-block p-3 text-light"> 
+                <i class="icon ion-md-contacts mx-2 lead"></i>
+                Manage Employees 
+            </a>
+            <a href="#" class="d-block p-3 text-light"> 
+                <i class="icon ion-md-stats mx-2 lead"></i>
+                Reports 
+            </a>
+        </div>
```
Comment (YilKanda) [inline checador/app/views/admin/dashboard.html.erb]: Check the indentation.

File: checador/app/views/admin/dashboard.html.erb [hunk 12]
```diff
@@ -1,9 +1,60 @@
...
+            <button class="btn btn-search position-absolute" type="submit"><i class="icon ion-md-clipboard"></i></button>
+          </form>
+          <ul class="navbar-nav">
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                Adminstrator
+              </a>
+              <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+                <li>
+                  <!--<a class="dropdown-item" href="#">Action</a>-->
```
Comment (YilKanda) [inline checador/app/views/admin/dashboard.html.erb]: If they don't occupy this line, delete it.

**Review by YilKanda — CHANGES_REQUESTED** [review]
Check the indentation of your files, since they use 2 spaces in some parts and 4 spaces in others. Remember that they must be uniform in the use of spaces (I recommend always using 2).

I also recommend that in the texts, instead of using plain text, they use [Rails Internationalization (I18n)](https://guides.rubyonrails.org/i18n.html), so that they can implement their application, not only in Spanish, but in several other languages ​​(for example, English), and in turn , allows them to have better control of their texts.

File: checador/app/assets/stylesheets/admin.scss (hunk 10 above)
Comment (xFerchoVJ) [reply checador/app/assets/stylesheets/admin.scss]: Okay, thank you, let me change it! :)

**Review by SEVERO96 — DISMISSED** [review]

**Review by JoseRicardoJimenez — DISMISSED** [review]
nice!

File: checador/app/views/layouts/application.html.erb (hunk 1 above)
Comment (YilKanda) [reply checador/app/views/layouts/application.html.erb]: It is recommended that you do not leave code in comments.

Remember that Git allows us to view this code on GitHub, in the commit in which it was created.

**Review by YilKanda — COMMENTED** [review]
It is recommended that you do not leave code in comments.

Remember that Git allows us to view this code on GitHub, in the commit in which it was created.

**Review by RobertoCarMed — DISMISSED** [review]

File: checador/app/views/layouts/application.html.erb (hunk 1 above)
Comment (xFerchoVJ) [reply checador/app/views/layouts/application.html.erb]: I'm going to use it later, thank you, Yil

**Review by SampabloC — DISMISSED** [review]

**Review by RobertoCarMed — APPROVED** [review]
LGTM!

**Review by SEVERO96 — APPROVED** [review]

**Review by SampabloC — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**